### DISCUSSION TITLE
Internet access seems to be required

### DISCUSSION THREAD

**Discussion by matthew-at-qamcom** [danny-avila/LibreChat#4307] [Help Wanted]
Hi,

Firstly, thanks for developing this beautiful front-end; it's incredibly professional.

I have set up LibreChat and got it working with my locally-hosted LLM (Llama 3.2 1B hosted by llama.cpp's server).

However, if I disconnect my laptop from the Internet, I can no longer refresh LibreChat.  All I see is a plain black screen.

My efforts at debugging this are:
* I see no errors in the Javascript console 
* I see no issues in the Network tab of the browser's developers' tools (all resources return a status of 200 from the cache)
* I see no errors in the Docker logs on any of the services
* I've waited several minutes and see no timeouts either (in Docker).

Is LibreChat's code just going in a never-ending loop trying to get access to a server on the Internet?  If so, what's it trying to do?  Are there changes I can make to the .env or librechat.yaml files to enable Internet-isolated functionality?

If it's useful, I'm running LibreChat v0.7.5-rc2.

Thanks greatly in advance,

Matthew

**Comment by pedrib** [comment]
Still a problem on LibreChat v0.7.8... @danny-avila any thoughts on this?
